lua/bind/cl_init.lua

```diff
@@ -165,6 +165,24 @@ function bind.FindAll(name)
 	return bt
 end
 
+---
+-- Whether or not this bind has been pressed and was not yet released.
+-- @param string name
+-- @return boolean
+function bind.IsPressed(name)
+	if not name then return end
+
+	local buttons = bind.FindAll(name)
+
+	for _, btn in ipairs(buttons) do
```

Comment:
A for struc would be faster than a ipairs / for-each struc ;)

Reply:
Done